### PR TITLE
93 assemble pipelines

### LINKED ISSUE #93: Execute/assemble interpretation pipelines selectively based on an input parameter
Implement a mechanism to specify which interpretation sub-pipelines are executed, for example: ALL (Default), Taxonomy, Geospatial, Temporary, etc. It must possible to specify 1 or more sub-pipelines to be executed. This mechanism must handles the configuration settings required by each sub-pipeline.

### PR DESCRIPTION
**Pull request by asturcon**
closes #93

### CODE REVIEW

File: beam-common/src/main/java/org/gbif/pipelines/config/EsProcessingPipelineOptions.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.DefaultValueFactory;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+
+@Experimental(Experimental.Kind.FILESYSTEM)
+public interface EsProcessingPipelineOptions extends DataProcessingPipelineOptions{
+
+  String[] DEFAULT_ES_HOSTS =
+    new String[] {"http://c4n1.gbif.org:9200", "http://c4n2.gbif.org:9200", "http://c4n3.gbif.org:9200",
```
Comment (fmendezh) [inline beam-common/src/main/java/org/gbif/pipelines/config/EsProcessingPipelineOptions.java]: we must remove these defaults

File: beam-common/src/main/java/org/gbif/pipelines/config/InterpretationType.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import com.google.common.collect.ImmutableList;
+
+/**
+ * Defines the types of the interpretations.
+ */
+public enum InterpretationType {
+
+  ALL, LOCATION, TEMPORAL, TAXONOMY, COMMON;
+
+  public static final List<InterpretationType> ALL_INTERPRETATIONS =
```
Comment (fmendezh) [inline beam-common/src/main/java/org/gbif/pipelines/config/InterpretationType.java]: use public static final EnumSet<InterpretationType> ALL_INTERPRETATIONS =
    EnumSet.of(LOCATION, TEMPORAL, TAXONOMY, COMMON); 
instead, why we need ALL_INTERPRETATION if we have an ALL literal

File: beam-common/src/main/java/org/gbif/pipelines/config/EsProcessingPipelineOptions.java (hunk 1 above)
Comment (asturcon) [reply beam-common/src/main/java/org/gbif/pipelines/config/EsProcessingPipelineOptions.java]: removed

File: beam-common/src/main/java/org/gbif/pipelines/config/InterpretationType.java (hunk 2 above)
Comment (asturcon) [reply beam-common/src/main/java/org/gbif/pipelines/config/InterpretationType.java]: changed

**Pull request merged**